### PR TITLE
Fix sanity test for modules

### PR DESCRIPTION
**Pull request by Akasurde**
##### SUMMARY

Signed-off-by: Abhijeet Kasurde <akasurde@redhat.com>

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
changelogs/fragments/misc_fix.yml
plugins/modules/acl.py
plugins/modules/at.py
plugins/modules/authorized_key.py
plugins/modules/seboolean.py
plugins/modules/selinux.py
plugins/modules/synchronize.py
plugins/modules/sysctl.py

### CODE REVIEW

**Comment by Akasurde** [comment]
Fixes CI failures - https://dashboard.zuul.ansible.com/t/ansible/build/004242c00fe2477981e4932817c60949/console

File: plugins/modules/synchronize.py [hunk 1]
```diff
@@ -165,7 +165,22 @@
     description:
       - Add a destination to hard link against during the rsync.
     type: list
-    default:
+    elements: str
+  _local_rsync_path:
```
Comment (felixfontein) [inline plugins/modules/synchronize.py:169]: I think these are used internally by the action plugin, and should probably not be documented.

Comment (Akasurde) [reply plugins/modules/synchronize.py:169]: ok

File: plugins/modules/synchronize.py [hunk 2]
```diff
@@ -141,12 +141,17 @@
       - Use the ssh_args specified in ansible.cfg. Setting this to `yes` will also make `synchronize` use `ansible_ssh_common_args`.
     type: bool
     default: no
+  ssh_args:
```
Comment (felixfontein) [inline plugins/modules/synchronize.py:144]: This one is provided by the action plugin as well (it depends on `use_ssh_args`). It probably should not be documented.

Comment (Akasurde) [reply plugins/modules/synchronize.py:144]: Yes. Sure.

**Comment by Akasurde** [comment]
CI failing due to [recent changes in selinux fact gathering](https://github.com/ansible/ansible/pull/73423) - fixed via https://github.com/ansible-collections/ansible.posix/pull/138

**Comment by Akasurde** [comment]
recheck

**Review by felixfontein — APPROVED** [review]

**Pull request merged**